internal/flink/utils.go

```
@@ -0,0 +1,52 @@
+package flink
+
+import (
+	"bytes"
+	"io"
+	"mime/multipart"
+	"net/http"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/dghubble/sling"
+)
+
+func uploadFile(url, filePath string, formFields map[string]any) error {
+	var buffer bytes.Buffer
+	writer := multipart.NewWriter(&buffer)
+
+	for key, value := range formFields {
+		if strValue, ok := value.(string); ok {
+			_ = writer.WriteField(key, strValue)
```

Why are we ignoring this error?